spec/board_spec.rb

```diff
@@ -0,0 +1,46 @@
+require './lib/board.rb'
```

Comment:
[Optional]
When you need to require a file it is best practice to use `require_relative`.